frontend/css/layout.css

```diff
@@ -66,13 +62,11 @@ a {
 .categories {
     justify-content: center;
```

Comment:
most of your styling that specifies `justify-content` and `align-items` is stating `center`, the difference i can see being under `.account`. 
So could that style be set more generally to avoid some repetition?  (Granted i'm only looking here at the diffs in this PR so it might not be the general case across _all_ the css).

Reply:
That's a good point. 

Would it be possible to apply the `container` class to the elements which currently have classes with the duplicated styles? 

I also added [a suggestion here](https://github.com/MinhazH99/orders101/pull/47/files#r1457830959) on how to specify the layout including child items of an outer container.